### PR TITLE
Fixes for long haul pipeline

### PR DESCRIPTION
**Pull request by gordonwang0**
Makes parameters for long haul similar to connectivity.

### CODE REVIEW

File: scripts/linux/trcE2ETest.sh [hunk 1]
```diff
@@ -875,7 +880,7 @@ elif [[ "${TEST_NAME,,}" == "${CONNECTIVITY_TEST_NAME,,}" ]]; then
     NETWORK_CONTROLLER_RUNPROFILE=${NETWORK_CONTROLLER_RUNPROFILE:-Offline}
 
     is_build_canceled=$(is_cancel_build_requested $DEVOPS_ACCESS_TOKEN $DEVOPS_BUILDID)
-    if [ $is_build_canceled -eq 1 ]; then
+    if [ "$is_build_canceled" -eq '1' ]; then
```
Comment (yophilav) [inline scripts/linux/trcE2ETest.sh:883]: is `"`  synonymous as `'` in bash?

File: scripts/linux/runE2ETest.sh [hunk 2]
```diff
@@ -15,6 +15,7 @@ function clean_up() {
 
     echo 'Remove IoT Edge and config file'
     rm -rf /var/lib/aziot/edged/
+    rm -rf /etc/systemd/system/aziot-edged.service.d/
```
Comment (yophilav) [inline scripts/linux/runE2ETest.sh:18]: We will likely need this in trcE2ETest.sh as well.

File: scripts/linux/trcE2ETest.sh (hunk 1 above)
Comment (gordonwang0) [reply scripts/linux/trcE2ETest.sh:883]: No. " interpolates, ' doesn't. So "$is_build_canceled" will be string value of $is_build_canceled and '1' is literal string '1'

File: scripts/linux/runE2ETest.sh (hunk 2 above)
Comment (gordonwang0) [reply scripts/linux/runE2ETest.sh:18]: Added

**Review by yophilav — APPROVED** [review]

File: scripts/linux/trcE2ETest.sh (hunk 1 above)
Comment (yophilav) [reply scripts/linux/trcE2ETest.sh:884]: Ok. I'm just making sure we are not comparing `1`  to `'1'`

**Pull request merged**